api/tests/property/include/wrapper.h

```diff
@@ -0,0 +1,8 @@
+#include "can_protocols/brake_can_protocol.h"
```

Comment:
`oscc.h` now includes the CAN protocol headers and `vehicles.h`, so you don't have to explicitly include these.

Reply:
881472f